Specification/BrAPI-Schema/BrAPI-Germplasm/Germplasm.json

```diff
@@ -462,17 +455,17 @@
                 }
```

Comment:
In v2.1 coordinates is a single object not an array. I think this is a mistake in the original schema @BrapiCoordinatorSelby ?

Reply:
Should I change/adjust anything here?
At least in the github overview, I don't see the correct place to which your comment refers.